themes/agnoster/agnoster.theme.sh

```diff
@@ -283,7 +283,8 @@ function prompt_end {
 
 ### virtualenv prompt
 function prompt_virtualenv {
-  if [[ -d $VIRTUAL_ENV ]]; then
+  # Exclude pyenv
+  if [[ -d $VIRTUAL_ENV && $PYENV_VIRTUALENV_INIT != 1 ]]; then
```

Comment:
After another look, I think maybe it is better to consistent with the condition in `promp_pyenv`, i.e., check also `pyenv`. With the current setup, nothing is shown when `PYENV_VIRTUALENV_INIT=1` is set but `pyenv` doesn't exist.

```suggestion
  # Exclude pyenv
  [[ $PYENV_VIRTUALENV_INIT == 1 ]] && _omb_util_binary_exists pyenv && return 0

  if [[ -d $VIRTUAL_ENV ]]; then
```